include/simdjson/simdjson.h

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 
+namespace SimdJson {
 struct simdjson {
```

Comment:
I would remove that struct. I think it was intended to play the role of a namespace or something. It will also prevent us from ending with stuff such SimdJson::simdjson::SUCCESS.